js/app/restaurant/components/Order/ChangeRestaurantOnEditFulfilmentDetailsModal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Modal from 'react-modal'
+import { useTranslation } from 'react-i18next'
+
+export default function ChangeRestaurantOnEditFulfilmentDetailsModal({
+  isWarningModalOpen,
+  setWarningModalOpen,
+}) {
+  const continueURL = window.Routing.generate('order_continue')
+
+  const { t } = useTranslation()
+
+  return (
+    <Modal
+      isOpen={ isWarningModalOpen }
```

Comment:
Would be nice to send a tracking event in Matomo analytics whenever the modal is open. 
Like [here](https://github.com/coopcycle/coopcycle-web/blob/78be4f99e4f26a411d1b04f829975c715514a21f/js/app/restaurant/components/DateModal.js#L25-L27)